boards/arm/ip_k66f/ip_k66f.dts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2020 DENX Software Engineering GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_k6x.dtsi>
+
+/ {
+	model = "SEGGER MK66F IP Switch board";
+	compatible = "nxp,mk66f18", "nxp,k66f", "nxp,k6x";
+
+	aliases {
+		led0 = &red0_led;
+		led2 = &red2_led;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		red0_led: led_0 {
+			gpios = <&gpioa 8 GPIO_INT_ACTIVE_HIGH>;
+			label = "User LD1";
+		};
+
+		red2_led: led_2 {
+			gpios = <&gpioa 10 GPIO_INT_ACTIVE_HIGH>;
+			label = "User LD2";
+		};
+	};
+};
+
+&cpu0 {
+	clock-frequency = <120000000>;
+};
+
+&flash0 {
+	reg = <0 0x200000>;
```

Comment:
Why is `reg` modified here?

Reply:
To adapt the flash size. I will create K66 dtsi for the flash size.

Reply:
Moved it to k66 dtsi. Thanks.